### PR TITLE
Web Vitals Docs

### PR DESCRIPTION
**Pull request by Steven0351**

### CODE REVIEW

File: website/docs/for-web/performance/ios/profiling.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  <img 
+    src={useBaseUrl('/img/profiling/ios/thumb/profiling-timeline-record-03.webP')} 
+    data-zoom-src={useBaseUrl('/img/profiling/ios/full/profiling-timeline-record-03.webP')}
+    width="75%"
+  />
+</em>
+
+### Native Http Caveat
+
+Due to a limitation in Safari Dev Tools, if the web application uses CapacitorHttp you will not get any insight into how those requests may be impacting your application performance in the timeline tab. Starting in Capacitor **insert version here**, a total time for the network request will be logged out to the console. This is not an issue for Android developers debugging their web applications in Chrome.
```
Comment (giralte-ionic) [inline website/docs/for-web/performance/ios/profiling.md:109]: we might want to indicate this is in a future version instead of 'insert version here'

**Review by giralte-ionic — CHANGES_REQUESTED** [review]
everything looks good, just want some better language about 'the future'

File: website/docs/for-web/performance/ios/profiling.md (hunk 1 above)
Comment (Steven0351) [reply website/docs/for-web/performance/ios/profiling.md:109]: This is available in 4.6.3 actually

**Review by giralte-ionic — APPROVED** [review]

**Review by carlpoole — APPROVED** [review]

**Review by jthoms1 — APPROVED** [review]

**Comment by carlpoole** [comment]
Looks good @jthoms1 - was wondering same as Stevens above comment

**Pull request merged**